aws/rust-runtime/aws-inlineable/src/presigning.rs

```diff
@@ -186,6 +186,20 @@ pub mod request {
                 .finish()
         }
     }
+
```

Comment:
let's also add something like `.with_body<B>(self, body: B) -> http::Request<B> { self.0.map(|_|B) }` as a direct route without needing to go through the builder